tests/queries/0_stateless/02956_rocksdb_bulk_sink.sql

```diff
@@ -0,0 +1,6 @@
+-- Tags: no-ordinary-database, no-fasttest
```

Comment:
Effectevely it's the same, but would work if we enabled rocksdb in fast tests and do not require a comment with the reason

```suggestion
-- Tags: no-ordinary-database, use-rocksdb
```